### ISSUE TITLE
Random Affine Transform Accepts Only a Tuple of 2 Instead of 6 - BUG

### ISSUE THREAD

**Issue by adildahlan** [fepegar/torchio#509]
I'm trying to apply a RandomAffine transform to a volume (C, H, W, D). 
What I'd like to achieve is a (random) rotation around the Z axis. 
The documentation (https://torchio.readthedocs.io/transforms/augmentation.html#randomaffine) for this says:
**degrees** – Tuple (a1,b1,a2,b2,a3,b3) (a1,b1,a2,b2,a3,b3) defining the rotation ranges in degrees.
But when I provide more than 2 elements for this it gives an error: "if degrees is not a single number it must be a sequence of len 2".  (I have the same issue also with translation...)
I hope you solve this bug...

**Comment by fepegar** [comment]
Hi, @adildahlan. Could you please follow the instructions requested in the "Bug Report" template? For example, some code would help.

In the meantime, we recently added an [example to rotate a volume around a specific axis](https://torchio.readthedocs.io/auto_examples/plot_video.html#sphx-glr-auto-examples-plot-video-py) (in this case, the volume represents a video).

**Comment by adildahlan** [comment]
Hi @fepegar ,
Many thanks for getting back to me,
The example you sent me (https://torchio.readthedocs.io/auto_examples/plot_video.html#sphx-glr-auto-examples-plot-video-py) is exactly what i need!
I downloaded the plot_video.ipynb file but I am still getting the same error when I run it (as it is without any changes). Probably it used to run in a previous version of TorchIO...
I attached below a screenshot
![image](https://user-images.githubusercontent.com/65319353/116239108-2edc0b00-a75a-11eb-9927-deda982c4105.png)

**Comment by adildahlan** [comment]
Maybe for the time being, until this bug gets fixed, I can use an old version of TorchIO that used to work? Do you know which version of TorchIO the example you sent me was written in?
Many thanks for your help

**Comment by fepegar** [comment]
What version of TorchIO are you using?

**Comment by adildahlan** [comment]
I am using torchio version 0.17.48

**Comment by fepegar** [comment]
> Do you know which version of TorchIO the example you sent me was written in?

The examples are run using the latest version after each new commit.

> I am using torchio version 0.17.48

That version is pretty old. Please upgrade and try again: `pip install --upgrade torchio`.

**Comment by adildahlan** [comment]
Hi @fepegar,

Many thanks for highlighting this. I tried updating torchio but it failed. After several trials (I attached a screenshot of the error I was getting). 

![image (1)](https://user-images.githubusercontent.com/65319353/116440444-6c1ec680-a848-11eb-98f5-feab1d80b06b.png)
![image](https://user-images.githubusercontent.com/65319353/116440448-6d4ff380-a848-11eb-8502-300e16ed7c46.png)

What worked is installing  'Deprecated', 'humanize', 'nibabel', 'numpy', 'scipy', 'torch', 'tqdm' (the dependancies of torchio as highlighted here https://github.com/fepegar/torchio/blob/master/setup.py) then installing torchio without dependancies using the following command pip install --no-deps torchio 

Thanks!!
